src/components/tree/phyloTree/helpers.js

```diff
@@ -128,9 +128,18 @@ function isWithinBranchTolerance(node, otherNode, distanceMeasure) {
     const tolerance = (node.shell.that.dateRange[1]-node.shell.that.dateRange[0])/50;
     return (getTraitFromNode(node, "num_date") - tolerance < getTraitFromNode(otherNode, "num_date"));
   }
-  /* Compute the divergence tolerance similarly to above. This uses the approach used to compute the
-  x-axis grid within phyotree, and could be refactored into a helper function. Note that we don't store
-  the maximum divergence on the tree so we use the in-view max instead */
+
+  /* divergence tree metric -- if there are any actual mutations
+  (i.e. not deletions / undeletions / gaps) then we are outside the tolerance */
+  const categorisedMutations = getBranchMutations(node, {}); // 2nd param of `{}` means we skip homoplasy detection
```

Comment:
> walk up the tree until a branch with mutations is found

Based on this comment, I was expecting to get mutations from `otherNode` here.

Reply:
Hmm, could you say more? The mutations on the branch between `otherNode` and `node` are encoded as a branch_attr on `node`

Reply:
Hmm, maybe I'm not understanding the underlying branch/node structure?

Within `getParentBeyondPolytomy`, the `potentialNode` is the only thing changing in the while loop, so I thought we had to check the branch mutations of the `potentialNode`. 

https://github.com/nextstrain/auspice/blob/07b66e80c61aea344b73f8c9aee12213a54c1a60/src/components/tree/phyloTree/helpers.js#L148-L154

Reply:
Huh. Now i'm confused as to how it worked when I tested it. What we want to do is check `node` first, then `node.parent`, then `node.parent.parent`, but it seems we're only ever checking `node`. I'll dive into this now. Thanks for the good 👀 !

Reply:
Ahh yeah this is broken in two places :( 

1. I assumed the both `node` and `otherNode` were updated each time through the loop, but they're not!
2. On branches without mutations, my code interpreted these as a tree without mutations and so it fell back to the old heuristic. 

I'll fix things up and re-push

Reply:
Fixed! @joverlee521 would you take a look at [2b3086b](https://github.com/nextstrain/auspice/pull/1552/commits/2b3086b8d0057ee897a4724d19efff3904b83cf5)? 🙏